src/sql/workbench/services/tableDesigner/browser/tableDesignerComponentInput.ts

```diff
@@ -76,10 +88,15 @@ export class TableDesignerComponentInput implements DesignerComponentInput {
 						errors: result.errors
 					}
 				});
+				editAction.withAdditionalMeasurements({
+					'elapsedTimeMs': new Date().getTime() - startTime
+				}).send();
 			},
 			error => {
 				this._notificationService.error(localize('tableDesigner.errorProcessingEdit', "An error occured while processing the change: {0}", error?.message ?? error));
 				this.updateState(this.valid, this.dirty);
+				this._adsTelemetryService.createErrorEvent(TelemetryView.TableDesigner, edit.type, error?.code,
+					error?.message).withAdditionalProperties(telemetryInfo).send();
```

Comment:
again, not sure where did my comment and Charles's comment go, please not use message here, it might contain sensitive information.

Reply:
please see previous thread for error messages